scripts/kinit_helper

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function kauth {
+    # If token doesn't exist, create it (will query for password)
+    if ! klist -s
+    then
+	while ! kinit
+	do
+            :
+	done
+    fi
+    return 0
+}
+
+function afsauth {
+    # afs needs kerberos token
+    kauth
+    # only available on psbuild servers
+    if [[ $(hostname) != psbuild-rhel* ]]
+    then
+        echo "You must be on psbuild to create afs tokens"
```

Comment:
This isn't quite true, but this is one of the few servers with the /afs mount point and this is where we should be doing the associated work, so the distinction is immaterial.